samples/app-cache-meetings/nodejs/src/components/app-cache-tab.tsx

```diff
@@ -0,0 +1,119 @@
+    readyToUnload: () => void) => {
+
+    let newItem = logItem("OnBeforeUnload", "purple", "Started");
+    setItems((Items) => [...Items, newItem]);
+
+    newItem = logItem("OnBeforeUnload", "purple", "Completed");
+    setItems((Items) => [...Items, newItem]);
+
+    console.log("sending readyToUnload to TEAMS");
+    readyToUnload();
+
+    return true;
+};
+
+/// </summary>
+/// loadHandler using setItems to set values
+/// </summary>
+const loadHandler = (
+    setItems: React.Dispatch<React.SetStateAction<string[]>>,
+    data: microsoftTeams.LoadContext) => {
+    logItem("OnLoad", "blue", "Started for " + data.entityId);
+
+    let newItem = logItem("OnLoad", "blue", "Completed for " + data.entityId);
+    setItems((Items) => [...Items, newItem]);
+
+    microsoftTeams.app.notifySuccess();
+};
+
+const AppCacheTab = () => {
+    const [items, setItems] = useState<string[]>([]);
+    const [title, setTitle] = useState("App Cache Sample");
+    const [initState] = useState(true);
+
+    React.useEffect(() => {
+        if (!initState) {
+            return;
+        }
+
+        microsoftTeams.app.initialize().then(() => {
+
+            microsoftTeams.app.getContext().then((context) => {
+                try {
+                    if (context.page.frameContext === "sidePanel") {
+                        const loadContext = logItem("Success", "green", "Loaded Teams context");
+                        setItems((Items) => [...Items, loadContext]);
+
+                        const newLogItem = logItem("FrameContext", "orange", "Frame context is " + context.page.frameContext);
+                        setItems((Items) => [...Items, newLogItem]);
+
+                        microsoftTeams.teamsCore.registerBeforeUnloadHandler((readyToUnload) => {
+                            const result = beforeUnloadHandler(setItems, readyToUnload);
+                            return result;
+                        });
+
+                        microsoftTeams.teamsCore.registerOnLoadHandler((data) => {
+                            loadHandler(setItems, data);
+                            setTitle("Entity Id : " + data.entityId);
+                            console.log(data.contentUrl, data.entityId);
+                        });
+
+                        const newItem = logItem("Handlers", "orange", "Registered load and before unload handlers. Ready for app caching.");
+                        setItems((Items) => [...Items, newItem]);
+                    }
+                }
+                catch (error) {
+                    console.log(error, "could not registered handlers");
+                }
+            });
+        });
+
+        return () => {
+            console.log("useEffect cleanup - Tab");
+        };
+
+    }, [initState]);
+
+    const jsx = initState ? (
+        <div>
+            <h3>{title}</h3>
+            {items.map((item) => {
+                return <div dangerouslySetInnerHTML={{ __html: item }} />;
```

Comment:
consider using FluentUI components like `<Text />` that you use in other samples instead here

Reply:
Using Simple HTML, Removed Fluent UI.

Reply:
This is the opposite of what i had requested! What's the reason? Are you not using FluentUI widgests in all samples?

Reply:
We are not using react NorthStar now as it is not supported in react version 18. We are using react component instead of React NorthStar and we couldn't find List component in react component. 